src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -777,6 +777,7 @@ singletonElement: emptyTag
             | metaTag
             | paramTag
             | wrongSinletonTag
+            | embedTag
```

Comment:
Do you say that for the readability of the whole thing or does it have something to do with how ANTLR resolves ambiguities ? If it is the former yeah it can be done, if it is the latter one, then I don't see it conflicting with the other rules below it, so it can be like this too. Also, as the ANTLR report in [4943#issuecomment](https://github.com/checkstyle/checkstyle/pull/4943#issuecomment-322046453), there doesn't seem to be any added ambiguities for the parser as the performance times have more less remained the same.